### ISSUE TITLE
Duplicated award icon

### ISSUE THREAD

**Issue by vadosnaprimer** [TASVideos/tasvideos#1987]
A user won the same movie award twice due to ties on 2 of their movies, and the icon for that movie award appears twice on their profile and posts.

https://tasvideos.org/Users/Profile/Cephla
https://tasvideos.org/Forum/Posts/527590

![изображение](https://github.com/user-attachments/assets/c7950d1a-8ab0-4594-b3e8-9d8244ba97e5)

**Comment by Masterjun3** [comment]
That's correct because they won the award with two movies, so they get two awards. See #767 .
